Bottom: 8e51ae702427b7a7a87950b153ed76720484dc1b
Top:    8a62e0f916c99daa6c2aa9c5d653b3e116b20ca2
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2016-04-29 01:43:43 -0500

Update README and convert to English


---

diff --git a/README.md b/README.md
index e3ade3f..99e20e6 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 tablecalc
 =========
 
-dokuwiki plugin nejen pro FO. Časem bude patrně nahrazen.
-
-Fork: https://www.dokuwiki.org/plugin:tablecalc
+Dokuwiki plugin which adds limited spreadsheet functionality to tables. See
+the original documentation page at: https://www.dokuwiki.org/plugin:tablecalc
 
+Forked from https://github.com/pirati-cz/tablecalc
